From 1cc17e3c8b1f73db43011775292396338f932a20 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui at cn.fujitsu.com>
Date: Mon, 17 Aug 2015 15:30:40 +0900
Subject: [PATCH] keyutils fix powerpc cflags

rebased the patch for powerpc arch,
it is required to resolve the patch application failure
with keyutils 1.5.9.

upstream-status: pending

Signed-off-by: Shrikant Bobade <shrikant_bobade@mentor.com>


Index: keyutils-1.5.5/Makefile
===================================================================
--- keyutils-1.5.5.orig/Makefile       2011-12-20 11:05:10.000000000
0200
+++ keyutils-1.5.5/Makefile    2011-12-20 11:06:27.000000000 +0200
@@ -58,1 +58,1 @@
 LNS		:= ln -sf

 ifeq ($(BUILDFOR),32-bit)
-CFLAGS		+= -m32
+#CFLAGS		+= -m32
 LIBDIR		:= /usr/lib
 USRLIBDIR	:= /usr/lib
 else
 ifeq ($(BUILDFOR),64-bit)
-CFLAGS		+= -m64
+#CFLAGS		+= -m64
 LIBDIR		:= /usr/lib
 USRLIBDIR	:= /usr/lib
 endif
-- 
1.8.4.2
